testsuite/tests/lib-unix/isatty/isatty.reference

```diff
@@ -0,0 +1,3 @@
+Unix.isatty Unix.stdin = false
+Unix.isatty Unix.stdout = false
+Unix.isatty Unix.stderr = false
```

Comment:
Please terminate this last line (and the corresponding printf) with a newline, it feels lonely without one.